### ISSUE TITLE
pgmagick compilation fails with graphicsmagick 1.3.24

### ISSUE THREAD

**Issue by kiefermat** [hhatto/pgmagick#8]
When trying to build and install pgmagick with graphicsmagick 1.3.24, the following error occurs:
```
./src/_Image.cpp:529:10: error: no matching member function for call to 'def'
        .def("xResolution", &Magick::Image::xResolution)
        ~^~~
/usr/local/include/boost/python/class.hpp:233:11: note: candidate template ignored: couldn't infer template argument 'F'
    self& def(char const* name, F f)
          ^
/usr/local/include/boost/python/class.hpp:223:11: note: candidate function template not viable: requires single argument 'visitor', but 2 arguments were provided
    self& def(def_visitor<Derived> const& visitor)
          ^
/usr/local/include/boost/python/class.hpp:242:11: note: candidate function template not viable: requires 3 arguments, but 2 were provided
    self& def(char const* name, A1 a1, A2 const& a2)
          ^
/usr/local/include/boost/python/class.hpp:249:11: note: candidate function template not viable: requires 4 arguments, but 2 were provided
    self& def(char const* name, Fn fn, A1 const& a1, A2 const& a2)
          ^
/usr/local/include/boost/python/class.hpp:265:11: note: candidate function template not viable: requires 5 arguments, but 2 were provided
    self& def(char const* name, Fn fn, A1 const& a1, A2 const& a2, A3 const& a3)
          ^
1 error generated.
error: command 'clang' failed with exit status 1
```
The error is reproducible both on a current Mac and on Debian Wheezy.

**Comment by hhatto** [comment]
fix c7f5b7093b05b09691020f09155acb2529535e8e , and [version 0.6.2](https://pypi.python.org/pypi/pgmagick/0.6.2) has been released.

Thanks.

**Comment by kiefermat** [comment]
Thanks a lot!
